test/unit/fake-document-mode.js

```diff
@@ -0,0 +1,7 @@
+QUnit.module( "fake-document-mode", { afterEach: moduleTeardown } );
```

Comment:
I haven't added this test to the list where all others are specified as it only makes sense in the mode with faked `documentMode`.